decidim-assemblies/spec/permissions/decidim/assemblies/permissions_spec.rb

```diff
@@ -534,7 +564,97 @@
           { scope: :admin, action: :list, subject: :assembly }
         end
 
-        it { is_expected.to eq(false) }
+        it { is_expected.to eq(true) }
+      end
+    end
+
+    describe "actions by assemblies types when user is an admin assembly" do
```

Comment:
Oh, I think I didn't explain myself well. This is really near to what I have in mind, just a couple of changes:
* Let's drop any mention to `AssemblyTypes` as it isn't related. That means that the block shouldn't be indented with assembly types (line 460), it should have its own first level block (at the same level of the “assembly types” one).  Can you move it out of 
* In line with that, it should be called "when acting with assemblies admins and children assemblies" - or something along these lines. 
For both of these cases (the indentation and the examples groups and examples descriptions), I use this command:

```bash
cd decidim-assemblies
bundle exec rspec -f d --dry-run ./spec/permissions/decidim/assemblies/permissions_spec.rb 
```

* We could also add other permissions here (like should it be able to destroy children assemblies?) but for now I'm OK with these as a starting point. If we start to see issues, we could always add more specs following these. 

So, can you change the indentation and the example group description 🙏🏽 ?

Reply:
Hi @andreslucena , thank you for your reviews!! Sorry, indentation it's my fault, I didn't realize of it.

I done the changes, can you review it again?